src/components/tile/_tile.scss

```diff
@@ -0,0 +1,168 @@
+//-----------------------------
+// Tiles
+//-----------------------------
+
+@import '../../globals/scss/colors';
+@import '../../globals/scss/layer';
+@import '../../globals/scss/typography';
+@import '../../globals/scss/import-once';
+@import '../../globals/scss/css--reset';
+@import '../../globals/scss/css--typography';
+
+@include exports('tile') {
+  .bx--tile-container {
+    width: 100%;
+
+    .bx--grid {
+      margin: 0;
+      padding: 0;
+    }
+
+    .bx--col {
+      background: none;
+      margin-bottom: 1rem;
+    }
+  }
+
+  .bx--tile {
+    @include layer('raised');
+    display: block;
+    width: 100%;
+    height: 100%;
+    min-width: 8rem;
+    min-height: 4rem;
```

Comment:
Some of the css properties here are a little bit clashing

`display:block` implies width: 100%, which is a little against `min-width: 8rem`. Are min-width/height's intended to be for demo purposes? Some of these styles might need to be moved away from the tile itself and into the demo.scss